CHANGELOG.md

```diff
@@ -24,8 +25,8 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [1.1.0] - 2017-05-20
 ### Added
-- Add `return-length` option to support custom length for returned matched lines
-- Add `log-pattern` option to support read and match against whole log entry instead of single line
+- Add `return-length` option to support custom length for returned matched lines (@jhshadi)
+- Add `log-pattern` option to support read and match against whole log entry instead of single line (@jhshadi)
```

Comment:
@majormoses I have tested it locally but i can add a unit test.
Can you reference to an example test and will take it from there?